tephra-hbase-compat-1.1-base/src/main/java/org/apache/tephra/hbase/coprocessor/TransactionProcessor.java

```diff
@@ -324,9 +326,14 @@ public InternalScanner preCompactScannerOpen(ObserverContext<RegionCoprocessorEn
                                       TxConstants.TransactionPruning.DEFAULT_PRUNE_ENABLE);
         String pruneTable = conf.get(TxConstants.TransactionPruning.PRUNE_STATE_TABLE,
                                      TxConstants.TransactionPruning.DEFAULT_PRUNE_STATE_TABLE);
-        compactionState = new CompactionState(c.getEnvironment(), TableName.valueOf(pruneTable));
-        LOG.debug("Automatic invalid list pruning is enabled. Compaction state will be recorded in table " +
-                    pruneTable);
+        long pruneFlushInterval = TimeUnit.SECONDS.toMillis(
```

Comment:
This line, and the line above getting `pruneTable` can also move into the if condition, right?